AsyncStreamReaderLoggingExample.py

```diff
@@ -0,0 +1,84 @@
+# Tomer Heber's coding challenge 20 Oct 2021.
+
+import asyncio
+import argparse
+import http.client
+import json
+import logging
+import time
+
+from abc import ABC, abstractmethod
+from datetime import datetime
+
+import httpx
+
+parser = argparse.ArgumentParser(description="Extracts Authentise events from the stream in an async manner.")
+parser.add_argument("--username", help="your authentise username", required=True)
+parser.add_argument("--password", help="your authentise password", required=True)
+args = parser.parse_args()
```

Comment:
please add (--base_url) for `dev-auth2.com` or `authentise.pro` etc . This is for the data going into {host} down below.

Reply:
added. If not configured will be set by default to dev-auth2.com